Tests/CombineTests/PublisherExtensionsTests.swift

```diff
@@ -0,0 +1,126 @@
+//
+//  PublisherExtensionsTests.swift
+//  SwifterSwift
+//
+//  Created by Heecheon Park on 7/28/23.
+//  Copyright © 2023 SwifterSwift
+//
+
+// swiftlint:disable identifier_name
+// swiftlint:disable large_tuple
```

Comment:
```suggestion
// swiftlint:disable identifier_name large_tuple
```

Reply:
Also, disable only for the functions that you need it, not the whole file.

Reply:
tbh, it was my first time using SwiftLint but I guess I get your point. Thanks,